tests/deploy/test_inferencer.py

```diff
@@ -23,11 +23,11 @@
 from omegaconf import DictConfig, ListConfig
 from pytorch_lightning import Trainer
 
-from anomalib.config import get_configurable_parameters
 from anomalib.data import get_datamodule
 from anomalib.deploy.inferencers import OpenVINOInferencer, TorchInferencer
 from anomalib.deploy.optimize import export_convert
 from anomalib.models import get_model
+from anomalib.utils.config import get_configurable_parameters
```

Comment:
Opening up yet another discussion. Should this be part of utils or stay in the main root level?